omnibus/config/projects/chef.rb

```diff
@@ -20,7 +20,7 @@
 homepage "https://www.chef.io"
 
 build_iteration 1
-build_version "12.6.0"
+build_version "12.7.0"
```

Comment:
we sort of have to use the Rakefile to keep the versions in the chef and chef-config gems synchronized, because they're published to rubygems and can't share a file.  that doesn't apply here, so i don't see any issue with using File.open and slurping in 'lib/chef/version.rb' since that should be authoritative.